### DISCUSSION TITLE
Meeting Types addition request

### DISCUSSION THREAD

**Discussion by Mickmous1** [code4recovery/12-step-meeting-list#1243] [Ideas]
I'm building my Al-Anon website and including TSML Version 3.14.23 for an Al-Anon Area in Canada. Wordpress Version 6.3.2.
In the "Types" checkboxes, English and Spanish are there. Could **French** be added in a future update?
We have some Bilingual English and French groups, as well as French speaking only groups in the area.
Never hurts to ask, right? Thank you for considering. Mickie

**Comment by tim-rohrer** [comment]
Hi Mickie,

Yeah, we surely can discuss this. For the AA program, the type `FR` is included for French.

The good news is that you don't have to wait for us! :-) Check out this [faq entry](https://wordpress.org/plugins/12-step-meeting-list/#my%20meeting%20type%20isn%27t%20listed%21) to see how to add your own custom type. You can do this with your site, and add other languages if appropriate.

Hopefully that helps!

Tim
